docs/content/en/integrations/parsers/file/hcl_appscan.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "HCL Appscan"
+toc_hide: true
+---
+Import HCL Appscan on cloud results. 
```

Comment:
Can you add a little more detail here about how to generate a report in the proper format for the parser? Doesn't have to be super detailed (see [Snyk](https://documentation.defectdojo.com/integrations/parsers/file/snyk/) and [Bandit](https://documentation.defectdojo.com/integrations/parsers/file/bandit/) examples)

Reply:
@ausmanki
could you help here?

Reply:
@cneill it need not be super detailed just the name of the vulnerability with the description and the fix to be implemented

Reply:
To ask this differently - is there any way to generate a report in HCL AppScan that would not produce a report like the one @manuel-sommer has implemented the parser to handle? For example, can you generate one in JSON format? If so, it would be good to tell the user what report format is expected and how to produce that report. If there is only a single way to generate a report, we don't necessarily need any further explanation.

Reply:
@ausmanki could you answer the question please? Then, we can merge this PR.

Reply:
apologies since I was travelling the report can be generated by the portal and can generate in XML pdf and csv